tests/test-delta.sh

```diff
@@ -249,6 +260,41 @@ ${CMD_PREFIX} ostree --repo=repo2 ls ${samerev} >/dev/null
 
 echo 'ok pull empty delta part'
 
+rm -rf repo/delta-indexes
+${CMD_PREFIX} ostree --repo=repo summary -u
+${CMD_PREFIX} ostree summary -v --raw --repo=repo > summary.txt
+assert_file_has_content summary.txt "ostree\.static\-deltas"
```

Comment:
Should also sanity-check here (and similarly below) that the directory is empty.

Reply:
Its not though, we always generate the delta indexes, even if they are *also* in the summary. This is because eventually we want to have some new version of the summary not have the deltas but the old one have it, and then new clients will use the delta indexes and old clients the deltast in the old summary.

Reply:
Ahh right OK, forgot about that.